src/components/Sidepane/EventDetails.js

```diff
@@ -48,6 +49,8 @@ class EventDetails extends React.Component {
     const timeString = this.getEventTimeString();
     const locationName = event.location_name || '';
     const locationString = event.address || 'Location unknown.';
+    //TODO: Get real userId
```

Comment:
@knguyend   Can this now be updated with Firestore access to the real uid? (so we can get rid of this TODO?)